src/open_company_web/components/new_report_popover.cljs

```diff
@@ -0,0 +1,75 @@
+(ns open-company-web.components.new-report-popover
+  (:require [om.core :as om :include-macros true]
+            [om-tools.core :as om-core :refer-macros [defcomponent]]
+            [om-tools.dom :as dom :include-macros true]
+            [om-bootstrap.random :as r]
+            [om-bootstrap.button :as b]))
+
+(def current-year (.getFullYear (new js/Date)))
+
+(def years (range 1950 (+ current-year 1)))
+(def periods ["M1"
```

Comment:
@bago2k4 also... just FYI, we are changing the product definition a bit so that weeks might become very interesting (W1, W2, W3... W52). So... we don't want to really write all 52 of those out by hand. Maybe play with generating this data, off a 1..4 and 1..12 range, so that it's easy to drop in W with a 1..52 range later if that's where we go.